src/Grids/Grids.jl

```diff
@@ -107,5 +107,6 @@ include("input_validation.jl")
 include("regular_rectilinear_grid.jl")
 include("vertically_stretched_rectilinear_grid.jl")
 include("regular_latitude_longitude_grid.jl")
+include("grid_timescales.jl")
```

Comment:
My comments below suggest moving this functionality into existing files. If we do keep this file, I think a more explicit name like `minimum_grid_spacings.jl` is better.